### ISSUE TITLE
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and when user press Shift+tab key, JAWS/NVDA does not announce anything: A11y_Azure Machine Learning Extension for VS Code_Appling filter on subscription_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [Microsoft/vscode-tools-for-ai#2495]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_Azure Machine Learning Extension for VS Code_Win32_Jan2024; #Azure Machine Learning Extension for VS Code; #WCAG1.3.1; #FTP; #Screen Reader; #NVDA; #JAWS; #Element:Combobox; #STP; #External; #External:VSCode; #FirstParty:Bug251068; #Regressed:08-08-25; #Closed; #A11yFixed;

## Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.0
Visual studio code version: 1.96.3
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS version: 2024.2312.53
NVDA Version: 2024.4.1

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Launch the VS Code.
3. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
4. Tab until the left toolbar (list of icons). This toolbar has a file icon, search icon, settings, etc.
5. Use the up/down arrow keys to navigate to the "Azure" icon and press enter
6. The Azure panel will now be active next to the toolbar.
7. Tab until the Machine Learning dropdown.
8. TAB till "Create job" button and press ENTER key.
9. Focus will be on "Select an Azure ML job Schema" combo box and.
10. Now press "TAB" than SHIFT + TAB.
11. Observer that JAWS/NVDA does not announce any thing when keyboard focus is on any list item of "Select an Azure ML job Schema" list and pressing shift +TAB key

## Actual Experience.
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and when user press Shift+tab key, JAWS/NVDA does not announce anything

### Similar issue observed with following scenarios.

**Scenario1**: Creating a job Select an Azure ML job Schema

1. TAB till create job button and press ENTER key
2. TAB till Combo box and press TAB and then SHIF + TAB

**Scenario2**: Setting Default Workspace

1. Open the Command Palette via the keyboard command Ctrl + Shift + P 
2. Type "Set default workspace" and select the option from the dropdown list called "Azure ML: Set Default Workspace") and press enter
3. A list should appear that says "Select a subscription"
4. Use the up/down arrows to navigate the list until "AML V1 Personal 2" is selected, and press enter
5. A list should appear that says "Select a workspace"
6. TAB till Select a workspace combo box and press TAB and then SHIFT + TAB.

## Expected Experience
When the keyboard focus is on any list item in the "Select an Azure ML job Schema" list and the Shift + TAB key is pressed, keyboard focus moved to "Select an Azure ML job Schema" combo box but JAWS/NVDA should announce about combo box. 

## User Impact:
Screen reader user will get confused if they press TAB key and screen reader is not announcing anything and keyboard focus is not visible and then pressing SHIFT + TAB also Screen reader is not announcing anything.

## Attachment

NVDA:

![Image](https://github.com/user-attachments/assets/ae851797-74af-428a-ba15-10f2fe0fee48)

JAWS:

![Image](https://github.com/user-attachments/assets/b4f40d5b-2a5c-4ff5-b3fd-009ae6699dd1)

Code snippet

![Image](https://github.com/user-attachments/assets/38f6de5a-dcc4-4a80-9944-dfd304331e88)

Similar issue of scenario 1 and 2

![Image](https://github.com/user-attachments/assets/2ca882e4-176d-4439-ba11-61fecb1b3b11)

**Comment by JerryYangKai** [comment]
Hello @kapilvaishna  I cannot repro the issue, I follow the step

Tab until the Machine Learning dropdown.
TAB till "Create job" button and press ENTER key.
Focus will be on "Select an Azure ML job Schema" combo box and.

Then NVDA will show 'Type to narrow down results' and the tittle, then the selections.
The screenshot

![Image](https://github.com/user-attachments/assets/a7b0465e-805b-4154-ba60-bbdb6337a233)

**Comment by kapilvaishna** [comment]
External bug: https://github.com/microsoft/vscode/issues/245224

**Comment by kupatkar99** [comment]
GithubTags:#ResolvedByTT;

**Comment by kapilvaishna** [comment]
External bug is fixed and as per external team  target release date is Aug 6th or Aug 7th for VS code.

**Comment by kapilvaishna** [comment]
Main and similar issues are not reproducing in VS code 1.103.0 so closing the bug.

Main issue fixed
https://github.com/user-attachments/assets/5311c00a-0556-4e50-ab3b-5ca96f6dffd0

Similar issue fixed

https://github.com/user-attachments/assets/634b4421-d4d5-4f80-b4a1-405c737da1d2
